src/foam/mlang/mlang.js

```diff
@@ -3223,16 +3223,7 @@ foam.CLASS({
   extends: 'foam.mlang.AbstractExpr',
   implements: [ 'foam.core.Serializable' ],
 
-  documentation: 'A Binary Predicate which applies arg2.f() to arg1.f().',
-
-  javaImports: [
-    'foam.core.AbstractFObjectPropertyInfo',
-    'foam.core.FObject',
-    'foam.core.PropertyInfo',
-    'foam.nanos.logger.Logger',
-    'foam.nanos.logger.StdoutLogger',
-    'foam.util.StringUtil'
-  ],
+  documentation: 'A Binary Expression which applies arg2.f() to arg1.f().',
```

Comment:
while you are here, can you also change this to
```suggestion
  documentation: `
    A Binary Expression which evaluates arg1 and passes the result to arg2.
    In other word, the output of arg1 is the receiver of arg2.
    
    For example, to get city from user address:
    
        DOT(User.ADDRESS, Address.CITY).f(user); // return user.address.city
        
    `,
```

Reply:
fixed